infra/fastly/terraform/dl-sandbox.k8s.dev/vcl/binaries.vcl

```diff
@@ -53,9 +53,8 @@ sub vcl_fetch {
 
   # TODO: Drop this when the origin(GCS bucket) is owned by the community
   # See: https://github.com/kubernetes/k8s.io/issues/2396
-  if (beresp.status == 200 && req.url.path ~ "^/release/") {
-    set beresp.http.Cache-Control = "private, no-store"; # Don't cache in the browser
-    set beresp.ttl = 30d;
+  if (beresp.status == 206 && req.url.path ~ "^/release/") {
```

Comment:
should we allow more status codes?

Reply:
🤔  I don't think we want to cache other responses from the origin for the migration. With a private bucket, we only get `206` from the origin based on some investigation with Fastly Support.

Reply:
OK, It seems like we might want to cache 2XX (at least 200??) just in case but we can revisit later